### PR TITLE
feat(e2e): adding playwright e2e tests for common Footer component

### PR DESCRIPTION
**Pull request by munzah-shah**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: e2e/playwright.config.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
+import path from 'path';
```
Comment (Ahmadkashif) [inline e2e/playwright.config.ts:1]: @munzah-shah
Can you explain why did you need to add this import?

Comment (munzah-shah) [reply e2e/playwright.config.ts:1]: I was facing the following problem:
The page.goto() method was not able to access the baseURL from the playwright config. Tested it quite a few times, it only passed the tests when I was passing the complete URL to the method for example: "http://localhost:8000/learn" and would fail on only "/learn". 
On asking @Fahdkassim and exploring the config file in his PR, he suggested me to add this snippet into the config file to fix the problem.
The .env file is not being read ig as it is not in the same directory as the config file, so this is just resolving the path to where the env file is located.

Note: this problem did not occur in my last PR tho.

**Review by Ahmadkashif — CHANGES_REQUESTED** [review]
The PR unfortunately has outlived its validity. Closing it for better clarity.
